src/main/steps/application-submitted/progress-bar.njk

```diff
@@ -0,0 +1,74 @@
+{% macro textWithVisuallyHiddenSpan(text, isComplete) %}
+  {{text}} <span class="govuk-visually-hidden">{{'' if isComplete else 'not '}}completed</span>
+{% endmacro %}
+
+{% if isJointApplication %}
+  {% set items = [{
+    label: {
+      html: textWithVisuallyHiddenSpan(submitted, true)
+    },
+    complete: true,
+    active: false
+  }, {
+    label: {
+      html: textWithVisuallyHiddenSpan(conditionalOrder, displayState.isAtOrAfter('AwaitingLegalAdvisorReferral'))
+    },
+    complete: displayState.isAtOrAfter('ConditionalOrderPronounced'),
+    active: hasApplicantConfirmedReceipt,
+    interval: weeksToConditionalOrder
+  }, {
+    label: {
+      html: textWithVisuallyHiddenSpan(finalOrder, displayState.state() == 'FinalOrderComplete')
+    },
+    complete: displayState.state() == 'FinalOrderComplete',
+    active: displayState.isAtOrAfter('AwaitingFinalOrder') and displayState.isBefore('FinalOrderComplete'),
+    interval: weeksToFinalOrder
+  }] %}
+{% else %}
+  {% set items = [{
+    label: {
+      html: textWithVisuallyHiddenSpan(submitted, true)
+    },
+    complete: true,
+    active: false
+  }, {
+    label: {
+      html: textWithVisuallyHiddenSpan(response, displayState.isAfter('IssuedToBailiff'))
+    },
+    complete: displayState.isAfter('IssuedToBailiff'),
+    active: displayState.isAtOrAfter('AwaitingAos') and displayState.isBefore('Holding'),
+    interval: weeksToResponse
+  }, {
+    label: {
+      html: textWithVisuallyHiddenSpan(conditionalOrder, displayState.isAtOrAfter('AwaitingLegalAdvisorReferral'))
+    },
+    complete: displayState.isAfter('ConditionalOrderPronounced'),
+    active: displayState.state() == 'AwaitingConditionalOrder' or hasApplicantConfirmedReceipt,
```

Comment:
@velimd there's a hasApplicantConfirmedReceipt here too

Reply:
good spot, i forgot i had 2 of these files